requirements.txt

```diff
@@ -29,3 +29,4 @@ google-cloud-pubsub
 oauth2client==3.0.0
 
 boto3
+pep8
```

Comment:
This would not be needed in production ?  Can we do a manual pip install inside the script instead ?

Reply:
How and when will check style get run ?

Reply:
I use flake8 for checkstyle; we can make it a part of tests, for example - or as a separate task in CI

Reply:
if it is a separate task, we don't necessarily need it in requirements.txt for the project, and we can install it only for the travis job